workalendar/asia/japan.py

```diff
@@ -54,6 +54,18 @@ def get_fixed_holidays(self, year):
             # Mountain Day is 8/10 this year for some reason
             # The next year that will be different is 2024
             days.remove((date(year, 8, 11), "Mountain Day"))
+        if year == 2021:
+            days.extend([
+                (date(year, 7, 22), "Marine Day"),
+                (date(year, 7, 23), "Sports Day"),
```

Comment:
Is it voluntary that the name of the holiday changed ?

Reply:
Also, the Sports day was a variable day, not a fixed day, so it should be patched in that method, no ?

Reply:
>Is it voluntary that the name of the holiday changed from Health and Sports Day to Sports Day?

No. The Japanese government changes the name to `Sports Day` in the law from 2020.

```
「スポーツの日」について
令和２年（2020年）以降、「体育の日」は「スポーツの日」になります。

国民の祝日に関する法律の一部を改正する法律（平成30年法律第57号）が平成30年6月20日に公布され、国民の祝日である「体育の日」の名称が「スポーツの日」に改められ、 その意義は「スポーツを楽しみ、他者を尊重する精神を培うとともに、健康で活力ある社会の実現を願う」とされました。（施行日：令和2年1月1日）
```
[Japanese Cabinet Office](https://www8.cao.go.jp/chosei/shukujitsu/gaiyou.html)

Reply:
>Also, the Sports & Marine Day day were variable days, not fixed days, so it should be patched in that method, no ?

They only change the date of the Sports & Marine Day this year(2021) because of the Tokyo Olympics. So I think they don't need to patched in the method.

Reply:
>Also, the Sports & Marine Day day were variable days, not fixed days, so it should be patched in that method, no ?

Yes, Should be patched in the method.

Reply:
> > Also, the Sports & Marine Day day were variable days, not fixed days, so it should be patched in that method, no ?
> 
> Yes, Should be patched in the method.

I change it in this commit 0a664455df7dd6de91777c23390ba00b1c9749be

Reply:
ah that's interesting. The main issue is that removing and adding tuples in the holiday list fails somehow because these days are not always added the right way.
I'm currently working on a branch that fixes this mishaps and also, I'm including a mechanism to set the correct Sport's Day label depending on the year.

Reply:
can you confirm that in 2020 the label was "Sports Day" or "Health & Sports Day"?

Reply:
Yes, in beginning in 2020, it renames to "Sports Day".

>Beginning in 2020, Health and Sports Day was permanently renamed Sports Day
(Wikipedia)[https://en.wikipedia.org/wiki/Health_and_Sports_Day]

Reply:
excellent! thank you very much!
incoming PR… -> #666